pants-plugins/bash/register.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Pants project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
```

Comment:
Would recommend namespacing this, as it might otherwise collide with https://pypi.org/search/?q=bash (as unlikely as that is...). Maybe `pantsbuild.examples.bash` ?

Reply:
That's only an issue if we use `bash` as a 3rd party library, right? If so, I think this is fine. Less nesting is easier to comprehend and we don't expect end-users to namespace with `examples` in their repo.

Reply:
It's an issue if any other plugin is using those libraries from pypi.

Reply:
+1 to namespacing this.

Reply:
Okay. I originally went with `pantsbuild`, but realized it's confusing what is `pants` vs. `pantsbuild`. So I'm going with `examples/bash`. Lmk if we should use a different name.